Modules/Core/Common/test/itkImageBaseGTest.cxx

```diff
@@ -98,6 +98,27 @@ Expect_by_default_Transform_result_equals_default_constructed_value(const typena
                                    CovariantVectorType());
 }
 
+
+template <unsigned int VImageDimension>
+void
+Check_New_ImageBase()
+{
+  const auto imageBase = itk::ImageBase<VImageDimension>::New();
+  ASSERT_NE(imageBase, nullptr);
+
+  for (const auto spacingValue : imageBase->GetSpacing())
+  {
+    EXPECT_FLOAT_EQ(spacingValue, 1.0);
+  }
```

Comment:
> Does this initialize the spacing to 1,1,1 or 1,0,0?

@dzenanz I interpreted your question as a request for an extra unit test, checking that the spacing of a new `ImageBase` object is indeed `{1, 1, 1}`.  Checked above here  😃

Reply:
While it wasn't necessary, computer does check it more reliable than us humans 🖥️👁️  😄